examples/api/patch_collection.py

```diff
@@ -42,6 +42,6 @@
 p = PatchCollection(patches, alpha=0.4)
 p.set_array(np.array(colors))
 ax.add_collection(p)
-plt.colorbar(p)
+fig.colorbar(p)
```

Comment:
fig.colorbar(p, ax=ax)? just so people have a frame of reference when they need multiple colorbars?